docs/reference/dockerd.md

```diff
@@ -1076,11 +1076,11 @@ The following is a full example of the allowed configuration options on Linux:
   "builder": {
     "gc": {
       "enabled": true,
-      "defaultKeepStorage": "10GB",
+      "reservedSpace": "10GB",
```

Comment:
afaics this is called `defaultReservedSpace`

Reply:
The struct doesn't set an explicit JSON name; does that mean it's technically with a capital `D`? Is that something we should change in moby to be explicit if we want it to be lowercase?